package.json

```diff
@@ -92,29 +98,24 @@
     },
     "peerDependencies": {
         "prop-types": "^15.5.10",
-        "react": "^15.0.1 || ^16.0.0",
-        "react-dom": "^15.0.1 || ^16.0.0"
+        "react": "^15.0.1 || ^16.0.0 || ^17.0.2",
+        "react-dom": "^15.0.1 || ^16.0.0 || ^17.0.2"
     },
     "scripts": {
         "commit": "git-cz",
         "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s",
         "build": "npm run clean && npm run compile",
         "build:es": "BABEL_ENV=es babel --copy-files ./src -d es",
-        "build:umd":
```

Comment:
Prettier got a hold of some of this. The only relevant changes are the deps above.